webapp/client/src/components/FormFieldDropDown.js

```diff
@@ -0,0 +1,101 @@
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import FormFieldScaffolding from "./FormFieldScaffolding";
+import FieldLabelForSeparatedFields from "./FieldLabelForSeparatedFields";
+import selectIcon from "../assets/icons/select_icon.svg";
+
+const DropDown = styled.div`
+  & label {
+    display: block;
+    width: 30px;
+    height: 30px;
+    cursor: pointer;
+    background: white;
+    position: absolute;
+  }
+
+  & .steuerlotse-select {
+    border: 2px solid var(--border-color);
+    border-radius: 0;
+    background-image: url(${selectIcon});
+    background-repeat: no-repeat;
+    background-size: 0.75rem;
+    min-height: 55px;
+  }
+
+  & .steuerlotse-select:focus {
+    border: 2px solid var(--focus-border-color);
+    box-shadow: 0 0 0 2px var(--focus-color);
+  }
+`;
+
+function FormFieldDropDown({
+  fieldName,
+  fieldId,
+  values,
+  defaultValue,
+  preselectedValue,
```

Comment:
What's the difference between preselected and default?

Reply:
I have renamed defaultValue to `defaultOption`. `defaultOption` Is a default option that is kept empty (like "Bitte auswählen"). The preselectedValue on the other hand would be a selection coming from the backend.

Do you see a better way for naming them or should we add a comment?

Reply:
Thanks! I understand `defaultOption` now. For prefilled data that comes from the backend, we use `value` in our other fields. `<select>`s are different though... Maybe a more consistent name could be `selectedOption`? On the other hand, React uses `defaultValue` to mark an option as selected on an uncontrolled select, so maybe `value` does work?

Reply:
Wait: `selectedValue`, not `selectedOption`, because `defaultValue` takes the value.

Reply:
This probably just confused you more! :)

Reply:
So,`defaultOption` and `selectedValue`?